content/en/docs/getting-started-guides/clc.md

```diff
@@ -252,7 +252,7 @@ kubectl cluster-info
 
 ### Accessing the cluster programmatically 
 
-It's possible to use the locally stored client certificates to access the apiserver. For example, you may want to use any of the [Kubernetes API client libraries](/docs/reference/client-libraries/) to program against your Kubernetes cluster in the programming language of your choice. 
+It's possible to use the locally stored client certificates to access the apiserver. For example, you may want to use any of the [Kubernetes API client libraries](/docs/reference/using-api/client-libraries/) to program against your Kubernetes cluster in the programming language of your choice. 
```

Comment:
Changing this file is OK; I won't hold up the PR over it. But if you continue to help fix broken links, I suggest skipping any files that don't appear in the sidebar TOC in the current build of the docs. (This includes much but not all of the content in /getting-started-guides/, which we're in the process of moving or deprecating.) I realize this makes checking for broken links more complicated, though. Another place you can look to see whether this content is included is in the _redirects file -- if a redirect is there, it means the topic is more likely to show up in the TOC.